docker-compose.yml

```
@@ -0,0 +1,37 @@
+    environment:
+      - NODE_ENV=production
+      - PORT=3000
+      # Add your Vercel access token here or via .env file
+      - VERCEL_ACCESS_TOKEN=${VERCEL_ACCESS_TOKEN}
+    ports:
+      - "3000:3000"
+    volumes:
+      # Mount config if needed
+      - ./config:/app/config:ro
+    restart: unless-stopped
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)